src/router/Router.js

```diff
@@ -9,7 +9,7 @@ function Router() {
       <Route path="/mockman" element={<MockAPI />} />
       <Route path="/login" element={<Login />} />
```

Comment:
`/` is not explicitly needed while defining routes for v6. You can omit those

Reply:
Thank you, I didn't know this, I've removed them now ^_^